src/svgs/icon-calendar-fill.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did we verify with @annemnichols that this icon is the correct one? Do we want to first strip it of metadata, and obfuscate? Not sure what our process is/should be here.

Reply:
This is the correct one (for now). Looks like the design team made designs for a new calendar icon but has not vetted it -- we might switch to the new one in the future. 

As for stripping, it's okay, in bigmaven we are using SVGo to optimize our SVGs and strip away most irrelevant data -- most of the metadata is from the Sketch library anyway and adding another step in the design process might be more headaches than SVGo. For our open-source aspect, we might just recommend using SVGo as well (or similar).